src/DistributedWriteAheadLog/KafkaWAL.cpp

```diff
@@ -273,14 +273,26 @@ void KafkaWAL::backgroundPollProducer() const
 void KafkaWAL::initProducerHandle()
 {
     std::vector<std::pair<std::string, std::string>> producer_params = {
+        /// Alias for `metadata.broker.list`
         std::make_pair("bootstrap.servers", settings->brokers.c_str()),
+        /// Maximum number of messages allowed on the producer queue.
         std::make_pair("queue.buffering.max.messages", std::to_string(settings->queue_buffering_max_messages)),
+        std::make_pair("queue.buffering.max.kbytes", std::to_string(settings->queue_buffering_max_kbytes)),
         std::make_pair("queue.buffering.max.ms", std::to_string(settings->queue_buffering_max_ms)),
         std::make_pair("message.send.max.retries", std::to_string(settings->message_send_max_retries)),
+        /// The backoff time in milliseconds before retrying a protocol request.
         std::make_pair("retry.backoff.ms", std::to_string(settings->retry_backoff_ms)),
+        /// the producer will ensure that messages are successfully produced exactly once
         std::make_pair("enable.idempotence", std::to_string(settings->enable_idempotence)),
+        // librdkafka statistics emit interval time.
```

Comment:
let's keep the comment in the same style.

Reply:
fixed